common/vcomputer-system.cpp

```diff
@@ -150,232 +141,229 @@ namespace tec {
-			if (this->computers.find(computer_itr->first) == this->computers.end()) {
-				this->computers[computer_itr->first] = computer_itr->second;
-			}
-		}
-		ProcessCommandQueue();
-		this->delta = _delta;
-		tda::TDAScreen screen;
-		//		static PixelBuffer local_pbuffer(320, 240, 8, ImageColorMode::COLOR_RGBA);
-		for (const auto& comp : this->computers) {
-			std::shared_ptr<ComputerScreen> comp_screen = std::static_pointer_cast<ComputerScreen>(comp.second->devices[5]);
-			comp.second->vc.Update(this->delta);
-			// #ifdef CLIENT_STANDALONE
-			// 			std::static_pointer_cast<tda::TDADev>(comp_screen->device)->DumpScreen(screen);
-			// 			tda::TDAtoRGBATexture(screen, (std::uint32_t*)local_pbuffer.LockWrite());
-			// 			local_pbuffer.UnlockWrite();
-			// 			if (comp_screen->texture) {
-			// 				comp_screen->texture->Load(local_pbuffer);
-			// 			}
-			// 			else {
-			// 				Entity screen_entity(comp.first);
-			// 				if (screen_entity.Has<Renderable>()) {
-			// 					Renderable* ren = screen_entity.Get<Renderable>();
-			// 					if (ren->buffer) {
-			// 						if (ren->buffer->GetVertexGroupCount() > 0) {
-			// 							comp_screen->texture = ren->buffer->GetVertexGroup(0)->material->GetTexture(0);
-			// 						}
-			// 					}
-			// 				}
-			// 			}
-			// #endif
+void VComputerSystem::Update(double _delta) {
+	EventQueue<KeyboardEvent>::ProcessEventQueue();
+	EventQueue<MouseBtnEvent>::ProcessEventQueue();
+	EventQueue<MouseClickEvent>::ProcessEventQueue();
+	EventQueue<EntityCreated>::ProcessEventQueue();
+	EventQueue<EntityDestroyed>::ProcessEventQueue();
+	for (auto computer_itr = ComputerComponentMap::Begin(); computer_itr != ComputerComponentMap::End();
+		 ++computer_itr) {
+		if (this->computers.find(computer_itr->first) == this->computers.end()) {
+			this->computers[computer_itr->first] = computer_itr->second;
 		}
 	}
+	ProcessCommandQueue();
+	this->delta = _delta;
+	tda::TDAScreen screen;
+	//		static PixelBuffer local_pbuffer(320, 240, 8, ImageColorMode::COLOR_RGBA);
+	for (const auto& comp : this->computers) {
+		std::shared_ptr<ComputerScreen> comp_screen = std::static_pointer_cast<ComputerScreen>(comp.second->devices[5]);
+		comp.second->vc.Update(this->delta);
+		// #ifdef CLIENT_STANDALONE
+		// 			std::static_pointer_cast<tda::TDADev>(comp_screen->device)->DumpScreen(screen);
+		// 			tda::TDAtoRGBATexture(screen, (std::uint32_t*)local_pbuffer.LockWrite());
+		// 			local_pbuffer.UnlockWrite();
+		// 			if (comp_screen->texture) {
+		// 				comp_screen->texture->Load(local_pbuffer);
+		// 			}
+		// 			else {
+		// 				Entity screen_entity(comp.first);
+		// 				if (screen_entity.Has<Renderable>()) {
+		// 					Renderable* ren = screen_entity.Get<Renderable>();
+		// 					if (ren->buffer) {
+		// 						if (ren->buffer->GetVertexGroupCount() > 0) {
+		// 							comp_screen->texture = ren->buffer->GetVertexGroup(0)->material->GetTexture(0);
+		// 						}
+		// 					}
+		// 				}
+		// 			}
+		// #endif
```

Comment:
can we delete this? I feel like we can delete this.

Reply:
Done